test.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
According to the [documentation](https://docs.ruby-lang.org/en/3.0.0/doc/syntax/comments_rdoc.html#label-frozen_string_literal+Directive) this directive should be used only if there are any string literals being specified in the file, otherwise there's no need to have it here.

Reply:
This file shouldn't have been included in the PR, it was me testing tests.